[#17] 기능 구현 및 React 동작 확인 완료

Byeong-soo opened the pull request:
### Issue_number
[#17]

# Summary

- [x] 그림판 가져오기 ( 너무 무거워서 안쓸듯)
- [x] 상대방 마우스 커서
- [x] 마우스 커서 밑에 닉네임 출력

# Check_List

- [x] Label 등록
- [x] Issue_number 등록
- [x] 불필요 파일 Commit 여부
- [x] Commit Message 규칙 확인

hoon25 committed:
암호키 변경 jks->p12
VIDU 테스트

Resolves : #14

Byeong-soo committed:
[#17] 기능 구현 및 React 동작 확인 완료

- 그림판 가져오기 ( 너무 무거워서 안쓸듯)
- 상대방 마우스 커서
- 마우스 커서 밑에 닉네임 출력

hoon25 committed:
[#14] WAS HTTPS -> HTTP, WAS urlMapping /api

Resolves : #14

hoon25 committed:
[#14] OpenVidu Video 채팅방 내부 동작 확인

css 변경 필요

Resolves : #14

Byeong-soo committed:
[#17] 마우스 포인터 위치 수정

- 화면 크기(해상도) 변경에 대한 위치 수정

hoon25 committed:
[#20] MySQL 연결정보 입력 및 jasypt 암호화 설정

Resolves : #20

hoon25 reviewed (commented):
PR 이름을 `기능구현` ->  `명확한 기능이름`을 써주시면 좋을것 같습니다.!

hoon25 committed:
[#14] OpenVidu Video 채팅방 내부 동작 확인

개선 필요 사항 - 비디오css, 퇴장 후 비디오 공간 유지

Resolves : #14

hoon25 committed:
[#14] Video 퇴장시 Subscribe 비동기 재밋는 이슈

콜백 비동기의 동작 과정이 신기해서 commit으로 남겨둡니다.
75(streamDestroyed), 31(deleteSubscriber)
코드가 실행을 미리하고, 적용을 콜백당시에 동작하게함

Resolves : #14

hoon25 committed:
[#14] Video 화면 동작

화상은 무제한 지원입니다.

Resolves : #14

Commit:
[#25] feat : 메인페이지 추가

hoon25 committed:
[#20] feat: Member관련 JPA 리팩토링, Spring 테스트코드 작성

유저관련 페이지 정상동작 확인
웹소켓때문에 단위테스트 불가능(통합테스트로 대체)
- 추후 웹소켓서버 분할 후 어노테이션 변경예정

Resolves : #20, #21

JacobKim0226 committed:
[#21] Tutor, Tutor찾기 API 설계

gender, grade, ranking 3 가지 항목에 따른 선생님 매칭
gender의 경우 BOTH 선택 시 남자, 여자 선생님 모두 찾기 가능
 * 추 후 회원가입 시 Tutor, Tutee 나눌 경우 Member 및 Tutor Entity 수정 예정

Byeong-soo committed:
[#17] 그림판, 및 Prettier 적용

- 기존에 있던 fabric 그림판 class component -> functional component 변경 완료.
- Prettier 적용 완료

hoon25 committed:
[#23] feat : Socket서버 초기 구축

환경변수 설정

resolves : #23

hoon25 committed:
[#23] feat : Socket서버 클라이언트와 정상 동작 확인

proxy : /sock  9001 port

resolves : #23

hoon25 committed:
[#23] feat : 기존 Spring WAS서버 Socket관련 코드 삭제

Resolves : #23

Byeong-soo committed:
[#17] 충돌 해결

- 충돌 해결

Byeong-soo committed:
[#17] 충돌 해결

- 충돌 해결

Byeong-soo committed:
[#17] feat : fabirc.js 동기화

- 객체 add 구현
- 해더파일 className 수정

Byeong-soo committed:
[#17] 충돌 해결

- 충돌 해결
